tests/benchmarks.py

```diff
@@ -0,0 +1,31 @@
+"""Benchmark tile."""
```

Comment:
we put the benchmark in a file without `test_` prefix so it's not run by pytest by default!